internal/cmd/update.go

```diff
@@ -36,14 +41,53 @@ func IsUnderHomebrew() bool {
 	return strings.HasPrefix(binary, brewBinPrefix)
 }
 
+var autoupdateCmd = &cobra.Command{
+	Use:   "autoupdate",
+	Short: "Manage your CLI autoupdate settings",
+}
+
+var autoupdateEnableCmd = &cobra.Command{
+	Use:               "enable",
+	Short:             "Enable autoupdates for the CLI",
+	Args:              cobra.NoArgs,
+	ValidArgsFunction: noFilesArg,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		settings, err := settings.ReadSettings()
+		if err != nil {
+			return fmt.Errorf("failed to read settings: %w", err)
+		}
+		settings.SetAutoupdate(true)
+		settings.SetLastUpdateCheck(time.Now().Unix())
```

Comment:
I would set it to 0 so that we actually do a version check once someone enables it.